README.md

```diff
@@ -1,61 +1,120 @@
+<p align="center">
+  <a href="https://www.pulumi.com?utm_campaign=pulumi-esc-github-repo&utm_source=github.com&utm_medium=top-logo" title="Pulumi ESC: Open source secrets management solution">
+    <img src="https://www.pulumi.com/images/logo/logo-on-white-box.svg?" width="350">
+   </a>
+
+  [![License](https://img.shields.io/github/license/pulumi/pulumi)](LICENSE)
+  [![Slack](http://www.pulumi.com/images/docs/badges/slack.svg)](https://slack.pulumi.com?utm_campaign=pulumi-esc-github-repo&utm_source=github.com&utm_medium=slack-badge)
+  [![X (formerly Twitter) Follow](https://img.shields.io/twitter/follow/PulumiCorp)](https://x.com/PulumiCorp)
+
 # Pulumi ESC (Environments, Secrets, and Configuration)
 
-Pulumi ESC is a new product from Pulumi that manages and tames secrets and configuration complexity across all of your cloud infrastructure and application environments. Pulumi ESC introduces a new category of configuration-as-code product, motivated by our experience working with hundreds of Pulumi IaC customers to address their needs in managing secrets and configuration at scale within their Pulumi infrastructure and across other cloud applications and infrastructure projects.
+**[Pulumi ESC](https://www.pulumi.com/product/esc/)** is an open source secrets management platform that tames secrets and configuration complexity across all of your cloud infrastructure and application environments.
+
+Pulumi ESC enables teams to aggregate secrets and configuration from many sources, manage hierarchical collections of configuration and secrets as environments, and consume those configuration and secrets from a variety of different infrastructure and application services.
 
-Pulumi ESC enables teams to aggregate secrets and configuration from many sources, manage hierarchical collections of configuration and secrets ("environments"), and consume those configuration and secrets from a variety of different infrastructure and application services.  Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project.
+Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project.
 
 For example, the Pulumi ESC CLI (`esc`) makes it possible to give your developers immediate, just-in-time authenticated and short-lived access to cloud credentials across any cloud provider with just a single command: `esc run aws-staging -- aws s3 ls`.
```

Comment:
The transition here from the previous paragraph doesn't feel smooth.

Reply:
Suggestions?

Reply:
(Needs proper formatting, grammar check and links but here is my suggestion)

[Pulumi ESC](https://www.pulumi.com/product/esc/) is an open-source secrets management platform that tames secrets and configuration complexity across all your applications and infrastructure. 

With Pulumi ESC, teams can aggregate secrets and configurations from many sources, manage hierarchical collections of configurations and secrets as `environments`, and consume them through a variety of means including CLI, SDK, REST API, Pulumi Cloud Web Console and Pulumi-service provider <add links to all of them>. 

Pulumi ESC not only works great for your applications and IaC, including Pulumi IaC, it also makes your day-to-day developer workflow much more secure and streamlined. For example, you Pulumi ESC CLI (esc) makes it possible to give your developers immediate, just-in-time authenticated and short-lived access to cloud credentials across any cloud provider with just a single command.

 $ esc run aws-staging -- aws s3 ls.
 
 In this command, an ESC environment named aws-staging has all the necessary staging environment configuration and OIDC setup to connect to AWS. Running this command opens up a temporary environment and executes the `aws s3 ls` command in that temporary environment. The temporary AWS credentials are not stored anywhere, making it secure and also allowing you to dynamically switch between different environments.

Reply:
thx